api/shipping_categories.rst

```diff
@@ -0,0 +1,424 @@
+Shipping Categories
+===================
+
+These endpoints will allows you to easily manage shipping categories. Endpoints' base is `/api/shipping-categories`.
+
+Shipping categories have two different serialization groups. While presenting a list you will receive an object with following fields:
```

Comment:
Remove first sentence.

When you get a collection of objects, "Default" serialization group will used and following fields exposed: